.buildkite/pipeline.yml

```diff
@@ -1,20 +1,24 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
 # Nodes with values to reuse in the pipeline.
 common_params:
   env: &xcode_image
     IMAGE_ID: xcode-14.3.1
   plugins:
-  - &docker_plugin
-    docker#v3.8.0:
-      image: &ruby_version "ruby:2.7.4"
-      propagate-environment: true
-      environment:
-        - "RUBYGEMS_API_KEY"
-  - &docker_plugin_with_danger_token
-    docker#v3.8.0:
-      image: *ruby_version
-      propagate-environment: true
-      environment:
-        - "DANGER_GITHUB_API_TOKEN"
+    - &ci_toolkit
+      automattic/a8c-ci-toolkit#2.18.2: ~
```

Comment:
This is the main change here — moving the plugin declaration from the step using it to it being at the top with the other anchors, + updating the version from `2.15.0` to `2.18.2`.

(The rest of the changes on this `common_params:` block are just fixing indentation)